### ISSUE TITLE
``Pileup`` hangs on WGS bam file

### ISSUE THREAD

**Issue by LeeTL1220** [broadinstitute/gatk#1920]
Command line from a gatk-protected jar file:

```
/dsde/working/lichtens/wgs$ gatk-protected/gatk-launch Pileup -R /seq/references/Homo_sapiens_assembly19/v1/Homo_sapiens_assembly19.fasta \
-I /seq/picard_aggregation/G23062/TCGA-BA-4077-01B-01D-2268-08/current/TCGA-BA-4077-01B-01D-2268-08.bam \
-L gatk_intervals_allchr.1kg.phase3.v5a.snp.maf10.biallelic.recode.fixed.prune5.interval_list \
-O test.txt
```

Uses `'org.broadinstitute:gatk:4.alpha.1-104-g213595d-20160606.150121-2'`

**Comment by magicDGS** [comment]
@LeeTL1220, I believe that it is because of huge coverage issues. Do you have any log? Does the output file contain something? I expect that the time for processing huge covered data will be slow because of the implementation. I'm sure that things will be better when downsampling is implemented.

**Comment by akiezun** [comment]
or was it NFS slowness. NFS is the worst.

**Comment by magicDGS** [comment]
If this was a coverage issue, downsampling argument is implemented in #2154. If not, should this be still open or this is a real issue? I'm interested because I'll use this part of the code in my work.

**Comment by droazen** [comment]
We should test to see whether the changes in https://github.com/broadinstitute/gatk/pull/2154 fixed this issue.
